package.json

```diff
@@ -160,11 +162,10 @@
     "react-popper": "1.3.11",
     "react-popper-2": "npm:react-popper@2.2.4",
     "react-resize-detector": "^9.1.0",
-    "react-windowed-select": "^5.2.0",
-    "smoothscroll-polyfill": "^0.4.4",
     "react-select": "^5.8.0",
-    "styled-system": "^5.1.4",
-    "@types/styled-system": "5.1.22"
+    "react-window": "^1.8.10",
+    "smoothscroll-polyfill": "^0.4.4",
```

Comment:
Unrelated to this PR, but I think we can remove this: https://caniuse.com/css-scroll-behavior

Reply:
I removed it.